package.json

```diff
@@ -26,6 +26,7 @@
     "antd": "^4.18.3",
     "axios": "^0.25.0",
     "classnames": "^2.3.1",
+    "framer-motion": "^6.2.8",
```

Comment:
Let's chat about the use of this. Every bundle adds to the build size and adds latency to the app... if this solves something that keyframes don't then great! If not we should just use keyframes.  Talk to me about what you'd like to achieve with this.